broker/client/client.go

```diff
@@ -675,7 +675,8 @@ func createSupplyingAgencyMessage(trCtx transactionContext, target *messageTarge
 
 func updateSupplierNote(trCtx transactionContext, sam *iso18626.SupplyingAgencyMessage) {
 	if trCtx.requester != nil && trCtx.requester.BrokerMode == string(common.BrokerModeOpaque) &&
-		(sam.StatusInfo.Status == iso18626.TypeStatusExpectToSupply || sam.StatusInfo.Status == iso18626.TypeStatusUnfilled) &&
+		(sam.StatusInfo.Status == iso18626.TypeStatusExpectToSupply || sam.StatusInfo.Status == iso18626.TypeStatusUnfilled ||
```

Comment:
Let's remove this check completely. It makes more sense to add this note to all messages.